frontend/src/index.tsx

```diff
@@ -59,9 +59,9 @@ createRoot(document.querySelector('#root') as HTMLElement).render(
                             {
                                 path: AppRoute.DASHBOARD,
                                 element: (
-                                    <PrivateRoute>
-                                        <Dashboard />
-                                    </PrivateRoute>
+                                    // <PrivateRoute>
+                                    <Dashboard />
+                                    // </PrivateRoute>
```

Comment:
Do we need to comment it out?

Reply:
Resolved in [8d68a7c](https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-smartspend/pull/244/commits/8d68a7ccec9fdd847e19084b1242fb42b087175e)